src/theme/icon.rs

```diff
@@ -45,7 +45,7 @@ impl Default for ByType {
             file: "\u{f016}".into(),         // 
             pipe: "\u{f731}".into(),         // 
             socket: "\u{f6a7}".into(),       // 
-            executable: "\u{f489}".into(),   // 
+            executable: "\u{f40d}".into(),   // 
```

Comment:
I feel like the newer ones look more like scripts than the one before because of the `<>`. Plus I think people might be used to this icon quite a bit by now and I don't want to make a change to a really common icon unless it is really required.

Reply:
> I feel like the newer ones look more like scripts than the one before because of the `<>`. Plus I think people might be used to this icon quite a bit by now and I don't want to make a change to a really common icon unless it is really required.

Hm, idk, I mean, I personally find the old icon weird, it makes every executable file a shellscript... Well, I'll remove it then